WORKSPACE

```diff
@@ -100,6 +100,14 @@ new_local_repository(
     workspace_file = "//third_party/rules_python:rules_python.WORKSPACE",
 )
 
+# Rules license must always be referred to as @rules_license even if imported
+# in to third_party. That allows the full classname of the providers to be
+# aligned across all rule sets.
+local_repository(
```

Comment:
Why not just use http_archive? We really shouldn't add more things in the third_party directory.